javascript/README.md

```diff
@@ -286,3 +295,137 @@ const { a, ...noA } = copy; // noA => { b: 2, c: 3 }
 - JSPerf: [Shallow Copy Objects][JSPerf: Shallow Copy Objects]
 
 [⇧ top](#javascript-guide)
+
+---
+
+## Strings
+
+### 5.1 Quotes
+
+Use single quotes `''` for strings.
+
+#### Examples
+
+🚫 Nope. 🚫
+
+```js
+const name = "Cloud Four";
+
+// Template literals should contain interpolation or newlines.
+const name = `Cloud Four`;
+```
+
+🎉 Yep! 🎉
+
+```js
+const name = 'Cloud Four';
+```
+
+#### Resources
+
+- ESLint: [quotes]
+
+### 5.2 Line Length
+
+Strings that cause the line to go over 80 characters should not be written across multiple lines using string concatenation.
```

Comment:
How does everyone feel about `80` vs `100`? I wouldn't mind if we set the `max-len` rule to `100`. But...I don't want to rock the boat too hard. 😅

Reply:
I didn't know where `80` came from. So I Googled it and found this article that kind of convinces me it's still a good idea: http://katafrakt.me/2017/09/16/80-characters-line-length-limit/

Reply:
(btw, not saying we should enforce it separately from Prettier or anything like that, just food for thought in a world where we _do_ specify it)

Reply:
Prettier also has an explanation for using 80 chars: https://prettier.io/docs/en/options.html#print-width

Reply:
@calebeby Do you feel this should also be a "Code should conform to Prettier formatting" instead?

Reply:
Yes

Reply:
I am removing stylistic sections from this guide for now.